index.html

```diff
@@ -4853,6 +4853,74 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to be usable to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Often relevant metadata about the <a>holder</a>, as identified by the value of the
+<code>holder</code> <a>property</a>, is available to, or
+retrievable by, the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is expected to be used when
+checking proofs on <a>verifiable presentations</a>. Some cryptographic
+identifiers contain all necessary metadata in the identifier itself. In those 
+cases, no additional metadata is required. Other identifiers use verifiable
+data registries where such metadata is automatically published for use
+by <a>verifiers</a>, without any additional action by the <a>holder</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by appropriate entities.  Validation is the process by which verifiers apply business rules to 
+evaluate the propriety of a particular use of a <a>verifiable credential</a>.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuers</a> define expressions of an `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> define expressions of a `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subjects</a> define expressions of a `credentialSubject` in <a>credentials</a>
+          </li>
+        </ul>
+        <p>
+A <a>verifier</a> might need to validate a given <a>verifiable presentation</a>
+against complex business rules; for example, the verifier might need confidence
+that the <a>holder</a> is the same entity as a <a>subject</a> of a <a>verifiable
+credential</a>. In that situation, the following factors can provide
+reasonable confidence that the holder presenting a given 
+<a>verifiable credential</a> is, in fact, a subject of that 
+<a>verifiable credential</a>:
```

Comment:
Working in @jandrieu's [suggestion, seen in full below](https://github.com/w3c/vc-data-model/pull/1199/files#r1310473764)
```suggestion
against complex business rules; for example, the verifier might need confidence
that the <a>holder</a> is the same entity as a <a>subject</a> of a <a>verifiable
credential</a>. In such a situation, the following factors can provide a
<a>verifier</a> with reasonable confidence that the claims expressed regarding
that identifier, in included <a>verifiable credentials</a>, are, in fact, about
the current presenter:
```